doom2016.asl

```diff
@@ -334,6 +405,17 @@ split {
             // Track to prevent splitting twice in 100%
             vars.visitedMapFiles.Add(current.mapFile);
             return true;
+        }  else if (!vars.hasSplitForIntro && settings["splitForIntro"]) { // Optional intro split when smashing the panel
```

Comment:
This is checking the last 2 patches, but this only works for the latest one right? You should check that here.